packages/hardhat-docker/package.json

```diff
@@ -13,8 +13,10 @@
     "docker"
   ],
   "scripts": {
-    "lint": "prettier --check \"**/*.{js,md,json}\" && tslint --config tslint.json --project ./tsconfig.json",
-    "lint:fix": "prettier --write \"**/*.{js,md,json}\" && tslint --fix --config tslint.json --project ./tsconfig.json",
+    "lint": "yarn prettier --check && yarn tslint",
+    "lint:fix": "yarn prettier --write && yarn tslint --fix",
+    "prettier": "prettier \"**/*.{js,md,json}\"",
+    "tslint": "tslint --fix --config tslint.json --project ./tsconfig.json",
```

Comment:
```suggestion
    "tslint": "tslint --config tslint.json --project ./tsconfig.json",
```

Reply:
Ouch, sorry I made so many silly mistakes in this PR.